addons/crm_iap_lead_enrich/views/crm_lead_enrich_views.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="crm_lead_enrich" model="ir.ui.view">
```

Comment:
Could you update according to guidelines ?
id: crm_lead_view_form
name: crm.lead.view.form.inherit.iap.lead.enrich
...
Filename: crm_lead_views.xml